Bottom: 896c3b2cfd19c79060220e0ac6e7a027190a8f3c
Top:    fe0d778f5f274071b409a4fb4092e1d93fc358f7
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-10-22 00:39:28 +0100

Refresh of try-something-with-the-rtrees

---

diff --git a/src/polygon1.c b/src/polygon1.c
index c958385..5741037 100644
--- a/src/polygon1.c
+++ b/src/polygon1.c
@@ -2246,7 +2246,6 @@ poly_InclContour (POLYAREA * p, PLINE * c)
       if (p->contours != NULL)
 	return FALSE;
       p->contours = c;
-      r_insert_entry (p->contour_tree, (BoxTypePtr) c, 0);
     }
   else
     {
@@ -2256,8 +2255,8 @@ poly_InclContour (POLYAREA * p, PLINE * c)
       tmp = p->contours->next;
       p->contours->next = c;
       c->next = tmp;
-      r_insert_entry (p->contour_tree, (BoxTypePtr) c, 0);
     }
+  r_insert_entry (p->contour_tree, (BoxTypePtr) c, 0);
   return TRUE;
 }
